kernel/core/io/file.cpp

```diff
@@ -4241,10 +4293,14 @@ namespace NumeRe
                             {
                                 styleId = style->IntAttribute("numFmtId");
 
-                                // Those are time formats: 14-22, 45-47
-                                if ((styleId >= 14 && styleId <= 22) || (styleId >= 45 && styleId <= 47))
+                                // Check if styleId is in vDateTimeIds set
+                                if (vDateTimeIds.find(styleId) != vDateTimeIds.end())
+                                {
+                                    // What change can i make to convertExcelTimeEpoch function?
                                     sValue = convertExcelTimeToEpoch(sValue, (styleId >= 18 && styleId <= 21)

```

Comment:
I am unsure about convertExcelTimeToEpoch. We haven't talked about this in detail yet, or much at all. It should be integral to date time functionality

Reply:
It is. It converts from 1.1.1900 to the UNIX epoch (i.e. 1.1.1970).